snapcraft/_baseplugin.py

```diff
@@ -75,11 +75,6 @@ def schema(cls):
             'build-properties': ['disable-parallel']
         }
 
-    @property
-    def PLUGIN_STAGE_SOURCES(self):
```

Comment:
Done, overriding property instead. I also left support for _PLUGIN_STAGE_SOURCES.